README.md

```diff
@@ -75,11 +77,13 @@ You will see:
 
 You can validate that Tach is working by either commenting out an item in a `depends_on` key in `tach.yml`, or by adding an import between modules that didn't previously import from each other. 
 
-Give both a try and run `tach check` again. This will generate an error:
+Give both a try and run `tach check` again. This will generate an error similar to this one:
 ```bash
-❌ path/file.py[LNO]: Cannot import 'path.other'. Tags ['scope:other'] cannot depend on ['scope:file']. 
+❌ tach/check.py[L8]: Cannot import 'tach.filesystem'. Tag 'tach' cannot depend on 'tach.filesystem'. 
```

Comment:
Do you think using `tach` as an example within `tach` is a bit too confusing? I do think the arbitrary one isn't great either.

One push I got was to update our readme with a motivating example - something for us to noodle more on.

Reply:
I think for now it's OK but agree we should have a more concrete, relatable example here.